src/frontend/components/LTINav/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { DASHBOARD_ROUTE } from '../Dashboard/route';
+import { PLAYER_ROUTE } from '../routes';
+import { appData } from '../../data/appData';
+import { Box, Button, Nav, Tab, Tabs, Text } from 'grommet';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { withRouter } from 'react-router';
+
+const messages = defineMessages({
+  linkDashboard: {
+    defaultMessage: 'Dashboard',
+    description: `Title for the dashboard, where the user can see the status of the video/audio/timed text upload
+      & processing, and will be able to manage them.`,
+    id: 'components.LTINav.linkDashboard',
+  },
+  linkPreview: {
+    defaultMessage: 'Preview',
+    description: `Title for the Instructor View. Describes the area appearing right above, which is a preview
+      of what the student will see there.`,
+    id: 'components.LTINav.linkPreview',
+  },
+});
+
+const NavItem = styled(NavLink)`
+  cursor: pointer;
+  padding: 1rem;
+  color: #444444;
+  text-decoration: none;
+  border-bottom: solid 2px #ededed;
+
+  :hover {
+    border-bottom-color: #007bff;
+  }
+
+  &.active {
+    border-bottom-color: #000000;
+  }
+`;
+
+const links = [
+  {
+    label: messages.linkDashboard,
+    route: DASHBOARD_ROUTE,
+  },
+  {
+    label: messages.linkPreview,
+    route: PLAYER_ROUTE,
+  },
+];
+
+export const LTINav = () => {
+  return (
+    <Box align="center" alignContent="center" pad="small">
+      <Nav direction="row">
+        {links.map((link) => (
```

Comment:
I don't know if using this this array is a good idea. For now all have `appData.modelName` as parameter in the `route` function but maybe not tomorrow. How to display link conditionally ?

Reply:
I think we could leave it like this for now, but I agree it could be improved.
I thought about adding per link permission checking there too.
But I don't like adding features before we really need them.
WDYT ?

Reply:
How will you manage the case where the preview should not be shown if the video is not ready to show ? Because I think in this case the dashboard link should be the only one displayed and selected. To know that you are on the dashboard.

Reply:
You will face an other problem. You depend on the resource state you are working on. This scenario will fail:

- I create a first LTI request, my object is created
- The dashboard link should only be shown
- I upload the resource
- Once the resource will be ready to be shown the `preview` link will not appear. I have to refresh the browser.

I think you should add props to your component to inject in it the current resource and access to the fresh state in the `LTINav` component.

Reply:
Do you think it's OK now ?